rskj-core/src/main/java/co/rsk/peg/BridgeSerializationUtils.java

```diff
@@ -801,15 +801,37 @@ public static byte[] serializeCoinbaseInformation(CoinbaseInformation coinbaseIn
     }
 
     public static byte[] serializeSha256Hash(Sha256Hash hash) {
+        if (hash == null){
```

Comment:
Can't this be wrapped on an alternative method? Checking if this kind of changes affects consensus and there's no need for them

Reply:
This is being fixed in this PR: https://github.com/rsksmart/rskj/pull/1928/files#diff-3bd14645f5f2bfbb02415a9c680c5c58e003143f285a357f760071d4ea0cd117